Ansible provisioning fails due to SELinux being in Enforcing mode

At the time of provisioning, docker build for build worker image fails with:

```
Sending build context to Docker daemon 54.78 kB
Step 1 : FROM registry.centos.org/centos/centos
Trying to pull repository registry.centos.org/centos/centos ... 
latest: Pulling from registry.centos.org/centos/centos
5ff99cd716ea: Pulling fs layer
5ff99cd716ea: Verifying Checksum
5ff99cd716ea: Download complete
5ff99cd716ea: Pull complete
Digest: sha256:a47ce555ff91eefbbbba20420a66ad3898a65b70bf5903d0ece6cee50af5d468
 ---> 9cfa2775d371
Step 2 : MAINTAINER CentOS Container Pipeline Service <container-group@centos.org>
 ---> Running in 18d329d2513a
 ---> f3fce1a0d6ba
Removing intermediate container 18d329d2513a
Step 3 : RUN yum -y install docker git PyYAML libyaml; yum clean all
 ---> Running in 338b8b091bfb
panic: standard_init_linux.go:178: exec user process caused "permission denied" [recovered]
	panic: standard_init_linux.go:178: exec user process caused "permission denied"

goroutine 1 [running, locked to thread]:
panic(0x6f3000, 0xc420118bd0)
	/usr/lib/golang/src/runtime/panic.go:500 +0x1a1
github.com/urfave/cli.HandleAction.func1(0xc420075748)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069a645069/Godeps/_workspace/src/github.com/urfave/cli/app.go:478 +0x247
panic(0x6f3000, 0xc420118bd0)
	/usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/opencontainers/runc/libcontainer.(*LinuxFactory).StartInitialization.func1(0xc420075198, 0xc42001e0c8, 0xc420075238)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/Godeps/_workspace/src/github.com/opencontainers/runc/libcontainer/factory_linux.go:259 +0x18f
github.com/opencontainers/runc/libcontainer.(*LinuxFactory).StartInitialization(0xc420047590, 0xaac9c0, 0xc420118bd0)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/Godeps/_workspace/src/github.com/opencontainers/runc/libcontainer/factory_linux.go:277 +0x353
main.glob..func8(0xc420078a00, 0x0, 0x0)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/main_unix.go:26 +0x66
reflect.Value.call(0x6ddd80, 0x769ce8, 0x13, 0x73c1c9, 0x4, 0xc420075708, 0x1, 0x1, 0x4d17a8, 0x732020, ...)
	/usr/lib/golang/src/reflect/value.go:434 +0x5c8
reflect.Value.Call(0x6ddd80, 0x769ce8, 0x13, 0xc420075708, 0x1, 0x1, 0xac2700, 0xc4200756e8, 0x4da786)
	/usr/lib/golang/src/reflect/value.go:302 +0xa4
github.com/urfave/cli.HandleAction(0x6ddd80, 0x769ce8, 0xc420078a00, 0x0, 0x0)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/Godeps/_workspace/src/github.com/urfave/cli/app.go:487 +0x1e0
github.com/urfave/cli.Command.Run(0x73c395, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x74d9d9, 0x51, 0x0, ...)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/Godeps/_workspace/src/github.com/urfave/cli/command.go:191 +0xc3b
github.com/urfave/cli.(*App).Run(0xc420096000, 0xc42000c120, 0x2, 0x2, 0x0, 0x0)
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/Godeps/_workspace/src/github.com/urfave/cli/app.go:240 +0x611
main.main()
	/builddir/build/BUILD/docker-96d83a5ff6ec0eb9bb7b45192c3048fd3aef5e21/runc-81b254244390bc636b20c87c34a3d9e1a8645069/main.go:137 +0xbd6
```

As suggested by @bamachrn this is because SELinux is set to Enforcing mode on the nodes. Changing this to Permissive should help resolve the issue.

--- a/provisions/vagrant.yml
+++ b/provisions/vagrant.yml
@@ -8,6 +8,10 @@
     - name: Enable centos admin repo when EPEL is disabled
       copy: src=centos-admin.repo dest=/etc/yum.repos.d/centos-admin.repo
       when: not enable_epel
+    - name: Set SELinux to Permissive
+      selinux:
+          policy: targeted
+          state: permissive
   tags:
     - jenkins
     - openshift

